docs/rules/valid-expect.md

```diff
@@ -5,7 +5,7 @@ Ensure `expect()` is called with a single argument and there is an actual expect
 ## Rule details
 
 This rule triggers a warning if `expect()` is called with more than one argument or without arguments. 
-It would also issue a warning if there is nothing called on `expect()`.
+It would also issue a warning if there is nothing called on `expect()` and when a matcher was not called.
```

Comment:
the wording here is a little weird, could you change this to "It will also issue a warning if there is nothing called on `expect()` or when a matcher function was not called."